types/auth.ts

```diff
@@ -24,3 +24,7 @@ export interface SessionContextProps {
 export interface QueryParams {
     [key: string]: string | string[];
 }
+
+export interface QueryProps {
+    [key: string]: string; // might be able to expand QueryParams interface
+}
```

Comment:
I believe GH is upset that there's not an empty line at the end of this file - there might be config in your IDE to automatically add this on save